lib/rubocop/cop/minitest/return_in_test_case.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Enforces the use of `skip` instead of `return` in test cases.
+      #
+      # @example
+      #   # bad
+      #   def test_something
+      #     return if condition?
+      #     assert_equal(42, something)
+      #   end
+      #
+      #   # good
+      #   def test_something
+      #     skip if condition?
+      #     assert_equal(42, something)
+      #   end
+      #
+      class ReturnInTestCase < Base
+        include MinitestExplorationHelpers
+        extend AutoCorrector
+
+        MSG = 'Use `skip` instead of `return`.'
+
+        def on_return(node)
+          return unless node.ancestors.any? { |parent| test_case?(parent) }
+          return if node.ancestors.find(&:block_type?)
```

Comment:
Is `numblock` the same?

Reply:
Thanks! Always forgetting this ruby feature.

Reply:
Thanks too :-) Can you add a test for that?

Reply:
Already added 😉

Reply:
Oops! I was overlooking. Thank you again!